### ISSUE TITLE
add support TypeScript

### ISSUE THREAD

**Issue by masayuki0220** [nuxt-community/nuxt-logrocket#99]
Thank you for providing nuxt-logrocket.

Can you add TypeScript support?

I have defined my own in my application as follows. I hope you can support ours officially.

```TypeScript
import LogRocket from 'logrocket'

declare module '@nuxt/vue-app' {
  interface Context {
    $logrocket: typeof LogRocket
  }
  interface NuxtAppOptions {
    $logrocket: typeof LogRocket
  }
}

// Nuxt 2.9+
declare module '@nuxt/types' {
  interface Context {
    $logrocket: typeof LogRocket
  }
  interface NuxtAppOptions {
    $logrocket: typeof LogRocket
  }
}

declare module 'vue/types/vue' {
  interface Vue {
    $logrocket: typeof LogRocket
  }
}
```

**Comment by farzadso** [comment]
Hi,

As I'm quite busy these days, do you think you could open a PR?

**Comment by masayuki0220** [comment]
Yes, I can open a pull request.

**Comment by farzadso** [comment]
TypeScript support is now available as of [`v1.3.0`](https://github.com/nuxt-community/nuxt-logrocket/releases/tag/v1.3.0).
